src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java

```diff
@@ -2895,4 +2895,88 @@ public void startFakeRepairSchedule() {
           Response.Status.NOT_FOUND);
     });
   }
+
+  @And("^I add 11 and abort the most recent 10 repairs for cluster \"([^\"]*)\" and keyspace \"([^\"]*)\"$")
+  public void addAndAbortRepairs(String clusterName, String keyspace) throws Throwable {
+    synchronized (BasicSteps.class) {
+      testContext.TEST_CLUSTER = (String) clusterName;
+      Set<String> tables = Sets.newHashSet();
+      testContext.addClusterInfo(clusterName, keyspace, tables);
+      HashMap<String, String> params = Maps.newHashMap();
+      params.put("clusterName", clusterName);
+      params.put("keyspace", keyspace);
+      params.put("owner", "test_user");
+
+      RUNNERS.parallelStream().forEach(runner -> {
+        Integer iter = 1;
+        while (iter <= 11) {
```

Comment:
You seem to be creating just 10 repairs here, not 11.
Also I don't think you should use a parallel stream across runners, because each runner will create 10 runs here.
Just pick the first runner by using `RUNNERS.get(0).callReaper(...)` at line 2913, and remove the surrounding stream.

Reply:
I was wondering about that. The runners all run against the same cluster? Changed as suggested.

Reply:
RE creating too few repairs, with this logic I create 1,2,3,4,5,6,7,8,9,10,11 - which is 11 I'm pretty sure? 

I used my fingers to count, so someone needs to rescind my math postgrad if I'm wrong about this.